composer.json

```diff
@@ -5,12 +5,19 @@
   "homepage": "https://github.com/localgovdrupal/localgov_subsites",
   "license": "GPL-2.0-or-later",
   "minimum-stability": "dev",
+  "repositories": [
```

Comment:
Hi James,
IIRC repository declarations only work in root-level composer.json files.  As a quick workaround, you can try updating the SQLite [authentication details](https://github.com/localgovdrupal/drupal-container/blob/596e0bc03241c240d1f7bf05ea5c53ba55a233ac/run-tests.sh#L31) used in the test runner Docker container.  Currently it says **sqlite://localhost/dev/shm/test.sqlite**.  Replacing it with **sqlite://database:database@localhost/dev/shm/test.sqlite** may work.  This will of course mean rebuilding the Docker container which Steve can do later.

I have to say I haven't tested this at all.  If you have SQLite in your development site, you should be able to try it there first.

FAO @iam-jam @stephen-cox